_posts/2024-05-31-friends-dont-let-friends-package-alone-pyconus-24.md

```diff
@@ -0,0 +1,282 @@
+transformed into what’s now the [pyOpenSci Python package guide](https://www.pyopensci.org/python-package-guide/).
+
+* A guidebook that has 58 contributors as I’m writing this post today.
+* A guidebook that has color and graphics and that leads Pythonistas down a
+  singular, opinionated path to Python packaging success.
+* A guidebook that breaks down pieces and processes into their fundamentals and
+  explains them using digestible words.
+
+The pyOpenSci path is not the only path. It is just a path that works and
+enables early wins for scientists and others using Python.
+
+If you're reading this and you have your own path, that's ok too. But if you
+want to worry less about decisions and have a definitive way of doing things
+that follows modern standards -
+
+we've got you.
+
+### A packaging guidebook created by community
+
+This guidebook was created by a community of people with beginner to expert packaging experience with the goal of building consensus
+around a single approach to pure Python packaging that would **yield user success**.
+
+Together we created a guidebook that acknowledges a suite of fantastic tools across the ecosystem while
+still leading users down a streamlined path.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/june/pyopensci-packaging-guide-overview.png" alt="Image with the pyOpenSci flower logo in the upper right hand corner. The image shows the packaging lifecycle. The graphic shows a high level overview of the elements of a Python package. the inside circle has 5 items - user documentation, code/api, test suite, contributor documentation, project metadata / license / readme. In the middle of the circle is says maintainers and has a small icon with people. on the outside circle there is an arrow and it says infrastructure." style="width: 70%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> The generalized scientific Python package development process. <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+
+## Getting accepted to talk about packaging...  <i class="fa-solid fa-ghost"></i>
+
+I was thrilled and terrified to be accepted to give a talk on the process of creating the pyOpenSci Python packaging guidebook at pyCon US in Pittsburgh, Pennsylvania. My talk was called:
+
+“Friends don’t let friends package alone”
+
+It was all about the process of navigating the Python packaging ecosystem and
+then creating the guide. While you might assume such a talk about be technical,
+in reality, it was all about people.
+
+### Me and the child-like imposter inside
+
+Giving this talk was a big deal for me. Imposter syndrome—the feeling that I
+didn't belong, that I wasn't technical enough to build pyOpenSci — has always been
+my reality. As a trained Landscape Architect, I spent most of my college and early graduate years
+practicing perfect lettering, using markers, fancy pens and colored pencils  to bring creative
```

Comment:
This should be a blog post and talk!!!

Reply:
imposter syndrome ? or the literal dozen of tubes with drawings that i can't let go of that i spent weeks on that are in my garage 😆 or both? i think the markers are all dried up.

i was terrible at lettering btw. never been good at handwriting. 😆

Reply:
ummmm I'd definitely read a post going through some of those tubes of drawings!